MMM-MBTA.js

```diff
@@ -613,6 +631,34 @@ Module.register("MMM-MBTA", {
         // Sorts them according to ETA time
         this.stationData.sort((a,b) => (a.preETA - b.preETA));
 
+        if (this.config.flipDirection && this.config.direction.length > 0)
+        {
```

Comment:
Please be consistent with the already existing guide. This should be

```
if (...) {
  ...
}
```

Thanks!

Reply:
There are a few more in the follow code too!